src/components/cards/CreatePool/ChooseWeights.vue

```diff
@@ -0,0 +1,213 @@
+<script setup lang="ts">
+import { configService } from '@/services/config/config.service';
+import TokenInput from '@/components/inputs/TokenInput/TokenInput.vue';
+import TokenWeightInput from '@/components/inputs/TokenInput/TokenWeightInput.vue';
+import useWeb3 from '@/services/web3/useWeb3';
+import { computed, onMounted, reactive, ref, nextTick } from 'vue';
+import useNumbers from '@/composables/useNumbers';
+import useBreakpoints from '@/composables/useBreakpoints';
+import anime from 'animejs';
+import { sum, sumBy } from 'lodash';
+import usePoolCreation, {
+  TokenWeight
+} from '@/composables/pools/usePoolCreation';
+
+const emit = defineEmits(['update:tokenWeights', 'nextStep']);
+
+const emptyTokenWeight: TokenWeight = {
+  tokenAddress: '',
+  weight: 0,
+  id: 0,
+  isLocked: false,
+  amount: 0
+};
```

Comment:
I mentioned this in Tim's PR but will repeat here. I think this type could be renamed, it's a little misleading since it contains more context than just the token weight. Maybe we could rename `NewPoolToken` or something similar?